src/styles/style.css

```diff
@@ -0,0 +1,122 @@
+@import url(./blocks/stars.css);
+
+html {
+  --accent: #060b35;
+  --text: #616070;
+  --link: #00acdc;
+  --cardborder: #f3f3f3;
+  --cardbgd: #fff;
+}
```

Comment:
I recommend moving the variables to a separate file.

Reply:
ok